src/ASM/ASMs1D.C

```diff
@@ -585,7 +585,7 @@ bool ASMs1D::updateRotations (const Vector& displ, bool reInit)
 }
 
 
-void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes) const
+void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes, int basis) const
```

Comment:
trenger ikke variabelen 'basis' her da vi vel ikke har støtte for mixed for 1D, dvs. det holder med "..., int) const"